### PR TITLE
Homework nodejs week1. Meals-sharing

### PR DESCRIPTION
**Pull request by MercedesUbeira**
Meal sharing - homework nodejs week1

### CODE REVIEW

File: src/backend/app.js [hunk 1]
```diff
@@ -21,6 +22,78 @@ app.use(cors());
 
 router.use("/meals", mealsRouter);
 
+app.get("/future-meals", async (req, res) => {
+  try {
```
Comment (LucasIversen) [inline src/backend/app.js:27]: `let now = new Date()` shouldn't be able to fail, so it can be put outside the try block

File: src/backend/app.js [hunk 2]
```diff
@@ -21,6 +22,78 @@ app.use(cors());
 
 router.use("/meals", mealsRouter);
 
+app.get("/future-meals", async (req, res) => {
+  try {  
+    let now = new Date();
+    let query = knex.from('Meal')
```
Comment (LucasIversen) [inline src/backend/app.js]: query and now can be made constants instead of variables

**Review by LucasIversen — APPROVED** [review]
Good job. SQL is spot on! In the app.js, my only comment would be to only put the code that might fail inside a try block, and to declare all your values as const, unless you want to update the value later on.

File: src/backend/app.js (hunk 1 above)
Comment (MercedesUbeira) [reply src/backend/app.js:27]: Thank you for all the feedback! I just made all the changes and made a new commit

**Pull request merged**